Merge changes from `2.5` into `2.6`

acolombier opened the pull request:
New content has landed in the `2.5` branch, so let's merge the changes into `2.6`

**NOTE:** This is a manual sync as there was currently conflict preventing the `sync_branch` workflow to open a PR. Shall we make it create an issue automatically to detect when this is happening?

~**NOTE 2**: Waiting feedback https://github.com/mixxxdj/mixxx/issues/15031#issuecomment-3015699141 and https://github.com/mixxxdj/mixxx/pull/15034 before merging to prevent subsequent merges.~

JoergAtGithub committed:
Switch GitHub runner to Windows-2022

JoergAtGithub committed:
Disabled 2 .aac.m4a test files on Windows, that always fail due to Win11 bug #11094

JoergAtGithub committed:
Add a check, if the MSVC toolset version is new enough

JoergAtGithub committed:
Updated README.md to VS2022 and made the name of the special shell bold

ronso0 committed:
Traktor Kontrol S4 Mk3: fix tempo fader offset (multiply with ticks/mm)

ronso0 committed:
Traktor Kontrol S4 Mk3: add per-deck tempo fader offset

acolombier committed:
chore: remove bare assert

daschuer committed:
Merge pull request #15006 from JoergAtGithub/update_to_windows2022_runner_image

Update 2.5 to windows2022 runner image

Swiftb0y committed:
Merge pull request #15000 from acolombier/chore/remove-bare-assert

chore: remove bare assert

acolombier committed:
Merge pull request #14882 from ronso0/s4-tempo-offset-per-deck

(fix/feat) Traktor Kontrol S4 Mk3: tempo offset per deck

daschuer committed:
Update Translation template. Found 3119 source text(s) (0 new and 3119 already existing)

daschuer committed:
Pull latest translations from https://www.transifex.com/mixxx-dj-software/mixxxdj/mixxx2-5/. Compile QM files out of TS files that are used by the localized app

ronso0 committed:
(fix) Pref Broadcast: make setting string translatable

Swiftb0y committed:
Merge pull request #15023 from ronso0/pref-broadcasting-tr-string

(fix) Pref Broadcast: make setting string translatable

acolombier committed:
ci(release): publish version to download server

Hapsa21 committed:
ci(2.5): remove all VCPKG_DEFAULT_HOST_TRIPLET entries from build.yml

acolombier committed:
ci(branch): allow running CI on fork branch without PR

acolombier committed:
Merge remote-tracking branch 'upstream/2.5' into sync-branch-2.5-to-2.6

daschuer committed:
Merge pull request #15029 from Hapsa21/fix/remove-unused-vcpkg-host-triplet

ci(2.5): remove all VCPKG_DEFAULT_HOST_TRIPLET entries from build.yml

JoergAtGithub commented:
@daschuer Should we merge this now into 2.6, or should we revert the mass reformatting in 2.5 before merge into 2.6? I'm unsure how to handle this.

daschuer committed:
Don't rename secrets when passing them to reusable workflows

This keeps workflows reusable workflows readable without following renaming in the caller. It also allows quickly using the work flow directly without renaming. It will be easier to maintain in case of renaming or adding secrets in the GitHub config.

JoergAtGithub committed:
Merge pull request #15034 from acolombier/chore/allow-running-ci-on-fork

ci(branch): allow running CI on fork branch without PR

daschuer committed:
Keep secret name MIXXX_BRANCH_SYNC_PAT unchanged

acolombier committed:
Merge pull request #15046 from daschuer/secret_renaming

Don't rename secrets when passing them to reusable workflows

acolombier reviewed (commented):
Waiting for ~#15049~ #15052

acolombier committed:
chore(sync-branches): fix regression preventing trigger

daschuer committed:
Update CHANGELOG for Mixxx 2.5.3

daschuer committed:
Merge pull request #15049 from acolombier/chore/fix-sync-branch

chore(sync-branches): fix regression preventing trigger

JoergAtGithub committed:
Merge pull request #15051 from daschuer/CHANGELOG25

Update CHANGELOG for Mixxx 2.5.3

acolombier committed:
chore(sync-branches): fix regression preventing trigger

daschuer committed:
Update Translation template. Found 3120 source text(s) (1 new and 3119 already existing)

daschuer committed:
Merge pull request #15052 from acolombier/chore/sync-branch-fix

chore(sync-branches): fix regression preventing trigger

acolombier committed:
Merge remote-tracking branch 'upstream/2.5' into sync-branch-2.5-to-2.6

acolombier commented:
Will merge if tests are successful, to unblock 2.6 beta snapshots.
